package.json

```diff
@@ -15,13 +15,15 @@
     "color": "^4.1.0",
     "colors": "^1.4.0",
     "cookie-parser": "^1.4.6",
+    "crypto": "^1.0.1",
```

Comment:
You actually didn't need to install `crypto` dependency. It is a Native Nodejs package ie it comes with Nodejs automatically. You need to require() it. That's all